uwblueprint/starter-code-v2
# create script to generate Firebase token

## Notion ticket link

[Create Firebase Token Script](https://www.notion.so/uwblueprintexecs/Create-Firebase-Token-Script-87724390f76f477fbe1bab6d21f817dc)



## Implementation description
Created a script to help developers generate the Firebase token quickly instead of having to generate it from the Postman. 



## Steps to test
1. From the CLI, run the command: "npm run dev -- --testing" and generate the PERN stack from "rohan/token-generator-script"
2. Build the application by running the command: "docker-compose up --build -d"
3. Run the following command from root of the repo: "npx ts-node token-generator.ts generate-token [email] [password]"
4. The script should give you an "accessToken". Copy and paste that accessToken and put it under “Authorization” of type “Bearer Token”
5. Make the necessary GET, POST, and DELETE requests
